README.md

```diff
@@ -19,4 +19,5 @@ For more information, please see the links below.
 
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python?ref=badge_large)
 
-test
+test2
```

Comment:
When making changes to the README, ensure that each modification improves the documentation or adds valuable information for users and contributors. Avoid committing test or placeholder content, especially in important files like README.md.